digitaltwins/samples/service/javascript/invoke_component_command.js

```diff
@@ -7,7 +7,7 @@ const DigitalTwinServiceClient = require('azure-iot-digitaltwins-service').Digit
 const deviceId = process.env.IOTHUB_DEVICE_ID;
 const connectionString = process.env.IOTHUB_CONNECTION_STRING;
 const componentName = process.env.IOTHUB_COMPONENT_NAME || 'sensor'; // for the environmental sensor, try "sensor"
-const commandName = process.env.IOTHUB_COMMAND_NAME || 'turnon'; // for the environmental sensor, you can try "blink", "turnoff" or "turnon"
+const commandName = process.env.IOTHUB_COMMAND_NAME || 'turnOn'; // for the environmental sensor, you can try "blink", "turnoff" or "turnon"
```

Comment:
the command in the comment should also be updated turnOff blink turnOn

Reply:
I'd rather delete de comment to avoid future maintenance